src/qililab/platform/platform.py

```diff
@@ -628,8 +628,18 @@ def execute_qprogram(
                     "Executing QProgram in more than one Quantum Machines Cluster is not supported."
                 )
             cluster: QuantumMachinesCluster = instruments.pop()  # type: ignore[assignment]
+            threshold_rotations = {
+                bus.alias: float(bus.get_parameter(parameter=Parameter.THRESHOLD_ROTATION))
+                for bus in buses
+                if isinstance(bus.system_control, ReadoutSystemControl)
+            }  # type: ignore
             return self._execute_qprogram_with_quantum_machines(
-                cluster=cluster, qprogram=qprogram, bus_mapping=bus_mapping, calibration=calibration, debug=debug
+                cluster=cluster,
+                qprogram=qprogram,
+                bus_mapping=bus_mapping,
+                threshold_rotations=threshold_rotations,  # type: ignore
```

Comment:
can this# type: ignore be solved by changing the _execute_qprogram_with_quantum_machines? shouldn't this be a dictionary or a None

Reply:
That is what I thought, the type for that argument in the method `_execute_qprogram_with_quantum_machines()` is indeed what you suggested but was still complaining so I added it.